docs/HowToAddAudioSupport.md

```diff
@@ -0,0 +1,62 @@
+
+# SECTION I: Tizen RT's Audio Framework
```

Comment:
# How to add audio support

## Audio Framework
blah blah

## How to add audio for a board
blah blah

Reply:
No. SECTION is needed here. Coz is has to be referred later.